### ISSUE TITLE
False positive Object.assing

### ISSUE THREAD

**Issue by gajus** [dustinspecker/eslint-plugin-no-use-extend-native#91]
```js
Object.assing(
  overwrite,
  _cfg.allChannels.allGeos.allTypes,
  _cfg.allChannels.allGeos[_ptype],
  _cfg.allChannels[_geo].allTypes,
  _cfg.allChannels[_geo][_ptype],
  _cfg[_channel].allGeos.allTypes,
  _cfg[_channel].allGeos[_ptype],
  _cfg[_channel][_geo].allTypes,
  _cfg[_channel][_geo][_ptype]
);
```

Here `Object.assing` is giving me an error `no-use-extend-native/no-use-extend-native`.

**Comment by dustinspecker** [comment]
Are you using `Object.assing` or `Object.assign`?

**Comment by gajus** [comment]
> Are you using Object.assing or Object.assign?

I am so sorry. :-(

**Comment by dustinspecker** [comment]
No worries! I'm glad to hear it's working fine.
